config/lint/.checkov.baseline

```diff
@@ -75,6 +75,28 @@
                 }
```

Comment:
There are a lot of policies added to this file. It's hard to review each exclusion.

Can you please keep only the ones about pinning digests (because we don't have a build infra for this example) for K8s stuff and Terraform stuff, so that we can check what to do with the others?

Reply:
This is still applicable